spec/example_app/app/models/blog.rb

```diff
@@ -0,0 +1,5 @@
+module Blog
+  def self.table_name_prefix
+    "blog_"
```

Comment:
I've not used namespaced models in quite a while. Is this necessary to override? Will be expecting all of our users to do the same?